web-ui/test/EditorPanelComponents/ExecuteButton.spec.js

```diff
@@ -0,0 +1,13 @@
+import { mount } from "@vue/test-utils";
+import ExecuteButton from "@/components/EditorPanelComponents/ExecuteButton";
+
+describe("ExecuteButton", () => {
+  test("Should emit a event with a click property", () => {
+    const wrapper = mount(ExecuteButton);
+    const button = wrapper.find(".executeButton");
```

Comment:
We should not test internal components, instead should just test the whole class:

```js
    const button = mount(ExecuteButton);
    button.trigger("click");
    expect(button.emitted().click).toBeTruthy();
```